OpenRA.Mods.Common/Warheads/HealthPercentageDamageWarhead.cs

```diff
@@ -22,6 +23,11 @@ public class HealthPercentageDamageWarhead : DamageWarhead
 		public override void DoImpact(WPos pos, Actor firedBy, IEnumerable<int> damageModifiers)
 		{
 			var world = firedBy.World;
+
+			var devMode = world.LocalPlayer.PlayerActor.TraitOrDefault<DeveloperMode>();
```

Comment:
This still crashes because `LocalPlayer` is `null`.
Put this whole thing in a `if (world.LocalPlayer != null)`.